src/main/resources/templates/multi-tool.html

```diff
@@ -57,24 +58,68 @@ <h2 th:text="#{multiTool.header}"></h2>
 
     <script>
         var fileName = null;
+
+        /**
+         * 
+         *  Manage Page container scroll
+         * 
+         */
         const pagesContainer = document.getElementById("pages-container");
+        const pagesContainerWrapper = document.getElementById("pages-container-wrapper")
 
-        // add the bottom "insert pdf" button that appears on the right when hovered over
-        const bottomInsertFileButtonContainer = document.createElement('div');
-        bottomInsertFileButtonContainer.classList.add("insert-file-button-container", "align-bottom");
-        bottomInsertFileButtonContainer.style.transform = "translate(0, 50%)";
-        
-            const bottomInsertFileButton = document.createElement('button');
-            bottomInsertFileButton.classList.add("btn", "btn-primary", "insert-file-button");
-            bottomInsertFileButton.innerHTML = `<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-file-earmark-plus" viewBox="0 0 16 16">
-                                    <path d="M8 6.5a.5.5 0 0 1 .5.5v1.5H10a.5.5 0 0 1 0 1H8.5V11a.5.5 0 0 1-1 0V9.5H6a.5.5 0 0 1 0-1h1.5V7a.5.5 0 0 1 .5-.5z"/>
-                                    <path d="M14 4.5V14a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V2a2 2 0 0 1 2-2h5.5L14 4.5zm-3 0A1.5 1.5 0 0 1 9.5 3V1H4a1 1 0 0 0-1 1v12a1 1 0 0 0 1 1h8a1 1 0 0 0 1-1V4.5h-2z"/>
-                                </svg>`;
-            bottomInsertFileButton.onclick = e => addPdfs();
-            bottomInsertFileButtonContainer.appendChild(bottomInsertFileButton);
+        const pageDirection = getComputedStyle(document.body).direction;
```

Comment:
Instead of computing the style you should be able to do something like
`document.documentElement.getAttribute("lang-direction")`